src/mmw/js/src/analyze/views.js

```diff
@@ -55,7 +55,30 @@ var ResultsView = Marionette.LayoutView.extend({
             modelPackageName = $(e.target).data('id'),
             modelPackage = _.find(modelPackages, {name: modelPackageName}),
             newProjectUrl = '/project/new/' + modelPackageName,
-            projectUrl = '/project';
+            projectUrl = '/project',
+            analysisResults = JSON.parse(App.getAnalyzeCollection()
+                                            .findWhere({taskName: 'analyze'})
+                                            .get('result')),
+            landResults = _.find(analysisResults, function(element) {
+                    return element.name === 'land';
+            });
+
+        if (landResults) {
+            var landAreaResults = landResults.categories,
+                landCoverTotal = _.sum(_.map(landAreaResults,
```

Comment:
Since `landAreaResults` is used only once here, and we're using `category` below, consider just using `landResult.categories` here and save an indirection.